File: 백준/Bronze/9498. 시험 성적/시험 성적.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){

        Scanner sc = new Scanner(System.in);
        int score = sc.nextInt();
        if(score % 10 != 0) score = score/10*10;
        switch(score){
            case 100 :
            case 90 :
            System.out.println("A");
            break;
            case 80 :
            System.out.println("B");
            break;
            case  70 :
            System.out.println("C");
            break;
            case 60 :
            System.out.println("D");
            break;
            default :
            System.out.println("F");
        }
    }
}

File: 백준/Bronze/9498. 시험 성적/시험 성적.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(){
    val input = BufferedReader(InputStreamReader(System.`in`))
    val score = input.readLine().toInt()
    when(score){
        in 90..100 -> println("A")
        in 80..89 -> println("B")
        in 70..79 -> println("C")
        in 60..69 -> println("D")
        else -> println("F")
    }
}